src/js_crypto/constants.ts

```diff
@@ -0,0 +1,328 @@
+// @gen this file is generated - don't edit it directly
+export { prefixes, versionBytes, poseidonParams }
+
+let prefixes = {
+  "event": "MinaZkappEvent******",
+  "events": "MinaZkappEvents*****",
+  "sequenceEvents": "MinaZkappSeqEvents**",
+  "body": "MinaZkappBody*******",
+  "accountUpdateCons": "MinaAcctUpdateCons**",
+  "accountUpdateNode": "MinaAcctUpdateNode**",
+  "zkappMemo": "MinaZkappMemo*******",
+  "signatureMainnet": "MinaSignatureMainnet",
+  "signatureTestnet": "CodaSignature*******"
+};
+let versionBytes = {
+  "tokenIdKey": 28,
+  "receiptChainHash": 12,
+  "ledgerHash": 5,
+  "epochSeed": 13,
+  "stateHash": 16,
+  "publicKey": 203,
+  "userCommandMemo": 20,
+  "privateKey": 90,
+  "signature": 154
+};
+let poseidonParams = {
```

Comment:
For clarity, and to avoid confusion, it would be good to name these according to our naming conventions here
https://github.com/o1-labs/proof-systems/tree/master/poseidon/src/pasta
For example, Kimchi or Legacy + Fp or Fq